core/src/main/java/org/elasticsearch/discovery/zen/ZenDiscovery.java

```diff
@@ -1313,4 +1329,9 @@ public void start() {
         }
 
     }
+
+    public final Collection<BiConsumer<DiscoveryNode, ClusterState>> getOnJoinValidators() {
```

Comment:
where is this used?

Reply:
I missed to `git add` one file that uses it. sorry for the noise... it's in a test